modules/shortcodes/js/jetpack-email-subscribe.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
+ */
+/**
+
+/* global jQuery */
+/* jshint esversion: 5, es3:false */
+var JetpackEmailSubscribe = {
+	validateEmail: function( email ) {
+		var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+		return re.test( String( email ).toLowerCase() );
+	},
+	activate: function( blogId, domId, cssPrefix ) {
+		var form = jQuery( '#' + domId );
+		form.submit( function( e ) {
+			e.preventDefault();
+			var emailField = form.find( '.' + cssPrefix + '-email' );
+			emailField.removeClass( cssPrefix + '-form-error' );
+			var email = emailField.val();
+
+			if ( ! JetpackEmailSubscribe.validateEmail( email ) ) {
+				emailField.addClass( cssPrefix + '-form-error' );
+				return false;
+			}
+
+			jQuery( '#' + domId + ' form' ).hide();
+			jQuery( '.' + cssPrefix + '-processing' ).css( 'display', 'block' );
```

Comment:
What's the difference between `.show()` and `.css( 'display', 'block' )`?

Reply:
`show` add animation

Reply:
done